randovania/game_connection/connector/prime1_remote_connector.py

```diff
@@ -114,6 +147,16 @@ async def _patches_for_pickup(
             if delta == 0:
                 continue
 
+            if item.extra.get("custom_value"):
+                patches.append(
+                    all_prime_dol_patches.adjust_item_amount_and_capacity_patch(
+                        self.version.powerup_functions,
+                        self.version.game,
+                        self.game.resource_database.get_item("Unknown2").extra["item_id"],
+                        item.extra["custom_value"],
```

Comment:
this needs to be the (id - 1000)
E.g. when we want to give missile launcher, we need to give 43, not 2